### ISSUE TITLE
Toggle Two-Pane View no longer works

### ISSUE THREAD

**Issue by hughpv** [a85/POSTMan-Chrome-Extension#13274]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

I can no longer toggle into two-pane view.

### Steps To Reproduce

1. Click the "View" menu
2. Click the "Toggle Two-Pane View" option

Expected: Should enter two-pane view
Observed: Nothing happens, but there is an error in the dev console:

```
common-651f0addde1f1ffe.min.js:1241 Uncaught TypeError: Cannot read properties of undefined (reading 'toggleLayout')
    at common-651f0addde1f1ffe.min.js:1241:46300
    at u.<anonymous> (common-651f0addde1f1ffe.min.js:1241:41563)
    at u.emit (vendors-83d9e33505e6bf68.min.js:1335:176491)
    at p (common-651f0addde1f1ffe.min.js:1233:4565)
    at EventEmitter.<anonymous> (common-651f0addde1f1ffe.min.js:1233:4655)
    at EventEmitter.emit (node:events:517:28)
    at Object.onMessage (node:electron/js2c/renderer_init:2:8536)
```

### Screenshots or Videos

https://github.com/user-attachments/assets/71d926b0-1aba-45da-b42f-67fc4ea6f535

### Operating System

macOS

### Postman Version

11.19.0

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by grantbob** [comment]
Seeing the same issue/error on with Postman App 11.20.0 on Windows 10.0.19045

**Comment by Harsh1796** [comment]
@hughpv @grantbob Thank you for reporting this issue. We have identified the root cause and will address it in an upcoming Postman release. We apologize for any inconvenience this may have caused. In the meantime, please use the toggle button in the bottom status bar to switch between one-pane and two-pane layouts.

<img width="523" alt="PostmanToggleTwoPaneButton" src="https://github.com/user-attachments/assets/53767f8d-ec68-4629-982a-3ed9e7bb55cc">

**Comment by Harsh1796** [comment]
@hughpv @grantbob We have rolled out a fix in Postman app version 11.22. Please update your application to the latest version. Let us know if the issue persists after updating.
